### PR TITLE
Minimist-fix: delete yarn.lock & package-lock.json

### PR DESCRIPTION
**Pull request by jchrisjones**
* ran npm update

* removed package-lock again

* ran npm install

* npm audit

* acorn versions 6.4.1 & 7.11

* minimist version 1.2.5

### CODE REVIEW

**Comment by ivanterziev-r3** [comment]
Hi @jchrisjones  @ilievZlatko can we close this today please if possible? See @barrylapthorn's comment above.

**Comment by barrylapthorn** [comment]
Sorry to be a pain, but `app.js` and `main.css` are now missing as far as I can see.

**Comment by ivanterziev-r3** [comment]
@jchrisjones @barrylapthorn Can we have this fixed today please?

**Comment by barrylapthorn** [comment]
Checked out branch and ran `hugo` looks good.

File: themes/hugo-r3-theme/webpack.config.js [hunk 1]
```diff
@@ -4,7 +4,8 @@ module.exports = [
     {
         entry: {
             app: ["./scripts", "./styles/scss/main.scss"],
-            vendor: "./scripts/vendor"
+            vendor: "./scripts/vendor",
+            "mermaid.min": "./node_modules/mermaid/dist/mermaid.min"
```
[the remark this answers is not in the document]
Comment (ivanterziev-r3) [reply themes/hugo-r3-theme/webpack.config.js:8]: @jchrisjones please remove this part and push and I'll merge - thanks!

**Pull request merged**